src/index.html

```diff
@@ -17,11 +17,139 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo Moyo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
```

Comment:
The `href` attributes for navigation links are set to '#', which doesn't lead anywhere. Consider providing valid URLs or implementing JavaScript functions to handle the click events if navigation is intended.